core/services/ocr2/plugins/ccip/evm/gas.go

```diff
@@ -0,0 +1,39 @@
+package evm
+
+import chainselectors "github.com/smartcontractkit/chain-selectors"
+
+const (
+	// DefaultExecTxGasOverhead is the default gas overhead for exec transactions. This value plus the offchainconfig.BatchGasLimit
+	// should be less than the gas limit of the chain. This is enforced by the offramp reader.
```

Comment:
Uber-nit: I find comments much easier to read and maintain if each sentence occupies one line, as go doc style guide recommends:

> The beginning of the comment is written using [semantic linefeeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/), in which each new sentence or long phrase is on a line by itself, which can make diffs easier to read as code and comments evolve.
-- https://go.dev/doc/comment

This way, adding or removing a few words in the middle of a sentence doesn’t result in multi-line diffs.

```suggestion
	// DefaultExecTxGasOverhead is the default gas overhead for exec transactions.
	// This value plus the offchainconfig.BatchGasLimit should be less than the gas limit of the chain.
	// Offramp reader enforces this invariant.
```